consent-ws/src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java

```diff
@@ -219,10 +222,11 @@ public Document describeSpecificFields(@PathParam("id") String id, @QueryParam("
     @Produces("application/json")
     @PermitAll
     public Consent describeConsentForDAR(@PathParam("id") String id) {
-        List<String> datasetId = (dataAccessRequestAPI.describeDataAccessRequestFieldsById(id, Arrays.asList(DarConstants.DATASET_ID))).get("datasetId", List.class);
+        List<Integer> dataSetIdList = (dataAccessRequestAPI.describeDataAccessRequestFieldsById(id, Arrays.asList(DarConstants.DATASET_ID))).get("datasetId", List.class);
```

Comment:
Note that in different objects, we have `"dataSetId"` vs `"datasetId"`. That refactor might be scope creep for this PR. I leave it to you as to whether we should do some renaming in this PR or address in a tech-debt story.

Reply:
Following up on the change from String to Integer ... legacy DARs that used the full SC-XXX number as the dataset ID causes this to throw a class cast exception. Adding one more reason I do not like how we're decomposing mongo objects ... lots of room for bugs here.

Reply:
See https://broadinstitute.atlassian.net/browse/DUOS-390 for fixing this.